desktop-app/app/actions/browser.js

```diff
@@ -672,6 +673,12 @@ export function screenshotAllDevices() {
   };
 }
 
+export function toggleEventMirroringAllDevices() {
+  return (dispatch: Dispatch, getState: RootStateType) => {
+    pubsub.publish(TOGGLE_EVENT_MIRRORING_ALL_DEVICES, [{now: new Date()}]);
```

Comment:
Why do you pass `[{now: new Date()}]` ? It is not been used right?

Reply:
I have simply copied below code snippet

```js
export function screenshotAllDevices() {
  return (dispatch: Dispatch, getState: RootStateType) => {
    pubsub.publish(SCREENSHOT_ALL_DEVICES, [{now: new Date()}]);
  };
}
```
I am not sure if its being used or not. Can you guide me on whether I should pass `now` or not

Reply:
the second argument of `pubsub.publish` is the info passed from the emitter to the handler function. In the case of `SCREENSHOT_ALL_DEVICES`, the date is needed to create a folder to store image files (the folder name includes date info)
In the case of `TOGGLE_EVENT_MIRRORING_ALL_DEVICES`, I think it is not needed so you can safely remove it